x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_extension.tsx

```diff
@@ -5,45 +5,461 @@
  * 2.0.
  */
 
-import React, { memo, useEffect } from 'react';
-import { EuiCallOut, EuiSpacer, EuiText } from '@elastic/eui';
+import React, { memo, useState, useCallback, useEffect } from 'react';
+import { EuiForm, EuiFlexGroup, EuiFlexItem, EuiCheckbox, EuiRadio, EuiSelect } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 import type { PackagePolicyCreateExtensionComponentProps } from '@kbn/fleet-plugin/public';
+import styled from 'styled-components';
 
 /**
  * Exports Endpoint-specific package policy instructions
  * for use in the Ingest app create / edit package policy
  */
+const SelectConfigurationSettingsText = styled.div`
+  font-size: 22px;
+  font-weight: 700;
+  padding-top: 12px;
+  padding-bottom: 10px;
+  line-height: 32px;
+`;
+
+const RadioOptionsDetails = styled.div`
+  padding-left: 24px;
+  color: #69707d;
+  font-weight: 400;
+  font-size: 12.25px;
+  line-height: 21px;
+`;
+
+const IntegrationOptionsContainer = styled.div`
+  margin-bottom: 48px;
+  margin-top: 17px;
+`;
+
+const CloudOptionDataIngestionMessage = styled.div`
+  font-size: 14px;
+  width: max-content;
+  font-weight: 400;
+  padding-top: 12px;
+  padding-bottom: 10px;
+  color: #69707d;
+`;
+
+const SubduedText = styled.div`
+  font-size: 14px;
+  font-weight: 400;
+  color: #69707d;
+  line-height: 24px;
+`;
+
+const BoldSubtitle = styled.div`
+  font-size: 14px;
+  font-weight: 700;
+  padding-top: 10px;
+  padding-bottom: 8px;
+`;
+
+const QuickSettingInfo = styled.div`
+  padding-bottom: 16px;
+  font-size: 14px;
+  line-height: 24px;
+`;
+
+const CloudRadioProtectionsModeContainer = styled.div`
+  padding-top: 8px;
+`;
+
+const DropDownSelect = styled(EuiSelect)`
+  margin-top: 14px;
+`;
+
 export const EndpointPolicyCreateExtension = memo<PackagePolicyCreateExtensionComponentProps>(
   ({ newPolicy, onChange }) => {
     // Fleet will initialize the create form with a default name for the integratin policy, however,
     // for endpoint security, we want the user to explicitely type in a name, so we blank it out
     // only during 1st component render (thus why the eslint disabled rule below).
+
+    // Run useEffect when page 1st load, default value are endpoint + NGAV
     useEffect(() => {
       onChange({
         isValid: false,
         updatedPolicy: {
           ...newPolicy,
           name: '',
+          inputs: [
+            {
+              _config: {
+                type: 'endpoint',
+                endpointConfig: {
+                  preset: 'NGAV',
+                },
+              },
+            },
+          ],
         },
       });
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    const dropDownOptions = [
+      { value: 'endpoint', text: 'Endpoint' },
+      { value: 'cloud', text: 'Cloud security' },
+    ];
+    const [dropdownValue, setDropdownValue] = useState(dropDownOptions[0].value);
+
+    const onChangeDropdown = useCallback(
+      (e: string) => {
```

Comment:
string  is not the correct type here, switch to `React.ChangeEvent<HTMLSelectElement>`